ror-tools/src/main/scala/tech/beshu/ror/tools/RorToolsApp.scala

```diff
@@ -122,29 +121,46 @@ trait RorTools {
     private def askUserAboutPatchingConsent(): PatchingConsent = {
       inOut.println("Elasticsearch needs to be patched to work with ReadonlyREST. You can read about patching in our documentation: https://docs.readonlyrest.com/elasticsearch#id-3.-patch-elasticsearch.")
       inOut.print("Do you understand the implications of ES patching? (yes/no): ")
-      inOut.readLine().toLowerCase match
-        case "yes" => PatchingConsent.Accepted
-        case _ => PatchingConsent.Rejected
+      inOut.readLine() match {
```

Comment:
That is the logic, that handles error with input being null (console input not possible)